packages/commons-server/src/libs/server/server.ts

```diff
@@ -197,6 +197,22 @@ export class MockoonServer extends (EventEmitter as new () => TypedEmitter<Serve
     }
   }
 
+  /**
+   * Set a global variable
+   */
+  public setGlobalVariables = (key: string, value: any): void => {
+    this.globalVariables[key] = value;
+  };
+
+  /**
+   * Resets the global variables
+   */
+  public purgeGlobalVariables = (): void => {
+    Object.keys(this.globalVariables).forEach(
+      (key) => delete this.globalVariables[key]
+    );
```

Comment:
This could be simplified to:
```suggestion
    this.globalVariables= {};
```